### DISCUSSION TITLE
Force speaker for missed readings alert

### DISCUSSION THREAD

**Discussion by tomascalta** [NightscoutFoundation/xDrip#3751] [Q&A]
Hello guys, is it possible to somehow set the missed readings alert to play only on phones speaker? For level alerts, there is an option called "Force speaker", but i cant find such option for missed readings alert and I already almost died one time because I left my bluetooth earbuds connected, missed readings alert didn't woke me up obviously and my pump kept delivering insulin throughout the night

**Comment by Navid200** [comment]
It has always been my concern that all xDrip alerts don't behave exactly the same:
https://github.com/NightscoutFoundation/xDrip/discussions/3697  

As you can see in that, it's not just the speaker.  The missed reading alert does not respect volume profile either.

It is my intent to make the alerts more consistent./  However, this is not easy.  That's why I still have not been able to do it.  
We need to use xDrip being aware of its limitations.  And we need to continue to improve xDrip.
I agree that we should add the setting you have asked for.

**Reply by tomascalta** [reply]
If I understand it correctly, you are doing all of this basically for free. Were you guys thinking about changing organization structure to allow Nightscout foundation to make some money, to employ some contributors and let them do it for money? For example, company behind OnlyOffice suit (opensource) runs some cloud services and that way they are able to push the development of OnlyOffice suit forward much faster. Im pretty new to this so excuse me if you guys already discussed this somewhere else.

**Reply by tomascalta** [reply]
I mean I am sure you thought about it, Im just curious why not go this way
